jest.config.js

```diff
@@ -9,4 +9,9 @@ module.exports = createConfig('jest', {
     'src/setupTest.js',
     'src/i18n',
   ],
+  moduleNameMapper: {
```

Comment:
What is the purpose of this? @AuraAlba

Reply:
This is for jest handle the asset files. This mock that because is not important in the process of testing: https://jestjs.io/docs/webpack